File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
     //have to return an integer
    //should return the length of the last word without space
    //string consist only letters and space
        String s1=s.trim();
        int count=0;
        for(int i=s1.length()-1;i>=0;i--){
            if(s1.charAt(i)!=' ')
                count++;
            else
                break;
        }
         return count;   
        }   
    }

File: 0058-length-of-last-word/0058-length-of-last-word.kt
class Solution {
    fun lengthOfLastWord(s: String): Int {
        val str = s.trim()

        for(i in str.length-1 downTo 0){
            if(str[i] == ' '){
                return str.length-i-1
            }
        }
        return str.length
    }
}